README.md

```diff
@@ -1 +1,36 @@
 # Choroscope
+
+## What is Choroscope?
+
+Choroscope is a visualization platform for geospatial data. Created to showcase results for the [Local Burden of Disease (LBD)](http://www.healthdata.org/lbd) project at the [Institute for Health Metrics and Evaluation (IHME)](http://www.healthdata.org), Choroscope provides a flexible format for exploring and sharing data linked to geography.
+
+## How does Choroscope work?
+
+The platform ingests a geospatial data set with an accompanying configuration file specifying the shape of the data and details of how it should be displayed. A database creation tool, `choroscope-db`, builds a geospatial database from the data, and an application server, `choroscope-api`, makes it available as a web API. A frontend web application, `choroscope-app`, uses this API to display both choropleths and highly detailed image-like maps - for which pixel colorization is done in the browser with a custom WebGL renderer. The configuration language, defined in `choroscope-spec`, allows the addition of other visualizations, like line and bar graphs, to supplement the primary map-based view.
```

Comment:
A simple architectural diagram with a few boxes showing the db, app, and spec would go a long way towards showing the different pieces and their relationships. I'm thinking you might have arrows and things that are labeled "reads", "builds", "configures". It would help me!

Reply:
Good idea! I'll think about this and see what I can come up with.

Reply:
Okay, diagram added.